build.gradle

```diff
@@ -3,7 +3,7 @@ import org.springframework.boot.gradle.plugin.SpringBootPlugin
 
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
```

Comment:
Isn't mavenCentral() there be default from Gradle itself @rpalcolea ?

Reply:
No, if you don't declare a repository, it won't resolve anything 😓